### PR TITLE
Added Spanish localizations for various UI strings.

### PR DESCRIPTION
**Pull request by tombogle**

### CODE REVIEW

**Comment by davyLSDev** [comment]
He revisado esto con mucho gusto.

File: DistFiles/Palaso.es.tmx [hunk 1]
```diff
@@ -486,6 +560,45 @@
...
+      <tuv xml:lang="es">
+        <seg>(La fuente seleccionada no está disponible en esta computadora. Se usará la fuente predeterminada.)</seg>
+      </tuv>
+    </tu>
+    <tu tuid="WSFontControl.RightToLeftWS">
+      <tuv xml:lang="en">
+        <seg>This is a &amp;right to left writing system.</seg>
+      </tuv>
+      <tuv xml:lang="es">
+        <seg>Este sistema de escritura se escribe de derecha a izquierda .</seg>
```
Comment (andrew-polk) [inline DistFiles/Palaso.es.tmx]: Space before the period

File: DistFiles/Palaso.es.tmx [hunk 2]
```diff
@@ -486,6 +560,45 @@
...
+      <tuv xml:lang="es">
+        <seg>&amp;Tamaño:</seg>
+      </tuv>
+    </tu>
+    <tu tuid="WSFontControl.TestArea">
+      <tuv xml:lang="en">
+        <seg>&amp;Test Area:</seg>
+      </tuv>
+      <tuv xml:lang="es">
+        <seg>Area de prueba:</seg>
```
Comment (andrew-polk) [inline DistFiles/Palaso.es.tmx]: No "&amp;" in the Spanish, though there is in English

File: DistFiles/Palaso.es.tmx [hunk 3]
```diff
@@ -391,6 +433,38 @@
 		<seg>Sí, mientras otros comparten por igual</seg>
 	  </tuv>
 	</tu>
+    <tu tuid="Project.CannotRemove">
+      <tuv xml:lang="en">
+        <seg>Cannot remove the selected project because it is currently open</seg>
+      </tuv>
+      <tuv xml:lang="es">
+        <seg>No se puede quitar el proyecto seleccionado porque está actualmente abierto.</seg>
```
Comment (andrew-polk) [inline DistFiles/Palaso.es.tmx]: Period in Spanish but not English

**Pull request merged**